Foundation/CharacterSet.swift

```diff
@@ -469,7 +469,7 @@ public struct CharacterSet : ReferenceConvertible, Equatable, Hashable, SetAlgeb
     
     /// Returns true if the two `CharacterSet`s are equal.
     public static func ==(lhs : CharacterSet, rhs: CharacterSet) -> Bool {
-        return lhs._wrapped.isEqual(rhs._bridgeToObjectiveC()) // TODO: mlehew - as  NSCharacterSet
+        return lhs._mapUnmanaged { l in rhs._mapUnmanaged { r in l.isEqual(r) } }
```

Comment:
Yep, reasonable, and makes sense. Comments both in here and in `_bridgeToObjectiveC()` explaining what makes this work (vs. doesn't) would be nice. Thank you! 🙂